decidim-core/app/controllers/decidim/pages_controller.rb

```diff
@@ -23,21 +23,18 @@ def page_finder
       @page_finder ||= Decidim::PageFinder.new(params[:id], current_organization)
     end
 
-    def users
-      @users ||= Decidim::User.where(organization: current_organization)
-    end
-
-    # This should be deleted once the statistics are done properly.
-    def participatory_processes
```

Comment:
It was but when I moved some logic to the presenter it didn't make sense to preserver that as a method so I just included inline this:

```ruby
def promoted_participatory_processes
      @promoted_processes ||= OrganizationParticipatoryProcesses.new(current_organization) | PublicParticipatoryProcesses.new | PromotedParticipatoryProcesses.new
    end
```